docs/DeveloperGuide.md

```diff
@@ -190,7 +183,7 @@ In this section we will explain how the `Automatic Tab Switching Feature` featur
  
 #### What is Automatic Tab Switching
 `Automatic Tab Switching` is a feature where the displayed tab automatically changes to the relevant tab: `Student
-` `Missions` `Quests` `Consultations` `Mastery Checks` and `Tasks`, for the user's input command.
+` `Missions`, `Quests`, `Consultations`, `Mastery Checks` and `Tasks`, for the user's input command.
```

Comment:
Thank you for the impeccable attention to detail.

Reply:
Impeccable attention to detail 👍

Reply:
Indeed so 👍